src/framework/constraints.ml

```diff
@@ -635,19 +669,165 @@ struct
       if M.tracing then M.traceu "combine" "combined local: %a\n" S.D.pretty r;
       r
     in
+    let handlelongjmp (cd,fc,fd) =
```

Comment:
Every normal function call also handles longjmps? Maybe I need to read the paper.

My concern is that this really complicates `FromSpec`, which is already complicated enough. I'm wondering if maybe it's possible to do this stuff in a functor.

Reply:
> Every normal function call also handles longjmps? 

It needs to - among others to account for modifications to locals that might have happened inside some function before the call to the function that finally executed the `longjmp`.